app/controllers/application_controller.rb

```diff
@@ -9,6 +9,9 @@ class ApplicationController < ActionController::Base
   before_action :set_phase_banner_text
   before_action :authenticate_basic
 
+  after_action :verify_policy_scoped,
+               if: -> { false && Rails.env.development? || Rails.env.test? }
```

Comment:
Is the `false &&` just for testing? 🤔

Reply:
Aaahh, ya. doh! I was just forcing it to test that path, good spot thanks!